ingest/src/main/java/com/connexta/ingest/client/StoreClient.java

```diff
@@ -39,7 +41,7 @@ public StoreClient(
 
   /** @return the location of the product */
   public URI store(
-      final Long fileSize,
+      @NotNull @Min(1L) @Max(10737418240L) final Long fileSize,
```

Comment:
`@NotNull` is included because
> `null` elements are considered valid.

for [`@Min`](https://javaee.github.io/javaee-spec/javadocs/javax/validation/constraints/Min.html) and [`@Max`](https://javaee.github.io/javaee-spec/javadocs/javax/validation/constraints/Max.html).